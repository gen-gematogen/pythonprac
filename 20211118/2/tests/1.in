class C:
        num = Num()

print(C().num)
c, d = C(), C()
c.num = d.num = 100
print(c.num)
print(d.num)
c.num = "Hello WOrld!"
print(c.num+d.num)
d.num = range(10, 100, 13)
print(c.num-d.num)
