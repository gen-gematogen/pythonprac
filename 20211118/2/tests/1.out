0
100
100
112
5
